Source/Tools/LinuxEmulation/LinuxSyscalls/Syscalls/NotImplemented.cpp

```diff
@@ -30,6 +30,8 @@ namespace FEX::HLE {
   // these are removed/not implemented in the linux kernel we present
 
   void RegisterNotImplemented(FEX::HLE::SyscallHandler *Handler) {
+      REGISTER_SYSCALL_NOT_IMPL(ustat);
+      REGISTER_SYSCALL_NOT_IMPL(sysfs);
```

Comment:
Looks like these were changed from placeholder implementations to fully unimplemented ones. Probably better to split this type of change into a separate commit for reference. (If this causes unexpected problems, it's easier to pin down the problem if it's not hidden in a 2000 line diff :) )

Reply:
Moved to a commit prior to the large one.